packages/core/src/components/type-ahead/type-ahead.scss

```diff
@@ -16,6 +16,7 @@
 }
 
 .typeahead-list-container {
+  position: absolute;
```

Comment:
Use of `absolute` seems to be an issue:

![image](https://github.com/UKHO/admiralty-design-system/assets/51328760/873a3222-8a2e-4d9f-bd59-3ab189e8f30f)

Maybe try `relative`?